sorc/build_all.sh

```diff
@@ -9,16 +9,6 @@ set +x
 #                   Anything other than "true"  will use libraries locally.
 #------------------------------------
 
-while getopts "c" option;
```

Comment:
I'm worried when this change gets merged into coupled-crow, it is going to remove the c option we need there.

Reply:
Want me to leave this line but still remove what that argument does?